index.html

```diff
@@ -128,6 +128,7 @@ <h2>
             <li><strong>Continuity</strong>: How to maintain a continuity of experience between immersive worlds?</li>
             <li><strong>Integration with the physical world</strong>: How to create persistent anchors and manage geographic alignment? How to capture and transmit the user's environment?</li>
             <li><strong>Integration with the web</strong>: How can regular web pages be rendered in immersive scenarios? How to navigate between immersive worlds through direct links?</li>
+            <li><strong>Intermediation</strong>: How can browsers maintain security and privacy boundaries in immersive world experiences? Can these boundaries be preserved to create intertwined experiences where users may carry personal artefacts managed by third-parties (such as a virtual smartphone) across immersive worlds?</li>
```

Comment:
```suggestion
            <li><strong>User Agent Intermediation</strong>: How can browsers maintain security and privacy boundaries in immersive world experiences, where the trust anchor provided by the browser UI is not necessarily easy to surface? Can these boundaries be preserved when using a browser in a virtual device in an immersive world?</li>
```
This adds bits about the UI as anchor of trust, and simplify the embedded question to browsing-in-metaverse.